### PR TITLE
Blog post about HYPR/Zendesk webinar

### PR DESCRIPTION
**Pull request by mooreds**
Wrote a post that hopefully will be useful beyond the webinar, but talks to why someone might want to attend.

### CODE REVIEW

File: site/_posts/2020-07-02-auth-and-the-bottleneck-architecture.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+* authentication - who is this user? 
+* authorization - what can this user do?
+* user management - registration, forgot password flows, two factor authentication, reporting, and more
+
+However, an auth system isn't limited to just verifying users locally. It can also federate to other providers of identity, such as Google, HYPR or Facebook. In addition, standards compliant providers adhering to the SAML or OIDC specifications can be integrated.
+
+Here's a diagram of the bottleneck architecture:
+
+{% include _image.liquid src="/assets/img/blogs/bottleneck-pattern/bottleneck-diagram.png" alt="A common archictural diagram." class="img-fluid" figure=false %}
```
Comment (voidmain) [inline site/_posts/2020-07-02-auth-and-the-bottleneck-architecture.md:33]: Total nit, but can we make the FusionAuth box our dark grey color and use the white icon with orange dots? :)

File: site/_posts/2020-07-02-auth-and-the-bottleneck-architecture.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+The applications, in purple, rely on the centralized auth system, which is in teal. They'll send all their auth requests to one place. Depending on how the auth system is configured, the applications' requests for user information may be answered directly, based on information in the auth system's datastore. Or, alternately, auth requests for a given user can be relayed to the federated identity providers, in orange. (You can read more about [the identity providers](/docs/v1/tech/identity-providers/) FusionAuth supports.)
+
+## Benefits Of a User Auth Service
```
Comment (voidmain) [inline site/_posts/2020-07-02-auth-and-the-bottleneck-architecture.md:37]: "Of" shouldn't be capitalized

File: site/_posts/2020-07-02-auth-and-the-bottleneck-architecture.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+
+If you work for a large organization, you may need to federate your user management system, using one of the standards such as OIDC, to other user datastores. Having a central service for your suite of applications means federation and integration only needs to be done once, rather than for each application you build.
+
+## Challenges
+
+Of course, nothing is perfect. There are challenges with this approach as well. Some technical, some not so much. 
+
+First off is the [SSO tax](https://sso.tax/). Many third party applications don't support this user auth delegation until you are on an enterprise plan. Ouch. Investigate required applications to see if and how they can act as a RP or SP to a centralized user management system before you decide to pursue this architecture. 
+
+Another organizational challenge is ensuring developers and users don't do an end run around the organization's user management system. Encourage employees to work within these constraints by making it as easy as possible and repeatedly explaining the benefits. Providing examples of successful integrations can help with both goals.
```
Comment (voidmain) [inline site/_posts/2020-07-02-auth-and-the-bottleneck-architecture.md:72]: I had to look up "end run around" and I think I get it. Maybe something a little more approachable?

Also, the first sentence says "developers and users", but this later talks about employees. Are "users" end-users or employees?

File: site/_posts/2020-07-02-auth-and-the-bottleneck-architecture.md [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+
+Finally, beware of insecure or slow auth services. No one cares about authentication and authorization, except when it doesn't work. When was the last time you heard someone exclaim "I love that login page!"? People want to log in when and how they choose and have it work, so they can get on to the task they need to accomplish. Select a system that is robust, has great support, and is flexible enough to meet future needs.
+
+## See It In Action
+
+If you'd like to see how easy it is to configure FusionAuth to serve as an centralized authentication and authorization service, HYPR and FusionAuth are hosting a webinar on Thursday, July 16 2020. 
+
+In less than 15 minutes, Zendesk will be set up as an application delegating auth decisions to FusionAuth (well, if you want to be technical, as an SP using SAML). HYPR will be configured as an identity provider. At the end, a user will be able to sign into Zendesk using HYPR's passwordless technology. [Sign up here!](https://get.hypr.com/fusionauth-webcast)
```
Comment (voidmain) [inline site/_posts/2020-07-02-auth-and-the-bottleneck-architecture.md:82]: Should we make the CTA a button rather than an inline link?

Comment (mooreds) [reply site/_posts/2020-07-02-auth-and-the-bottleneck-architecture.md]: Button made.

File: site/_posts/2020-07-02-auth-and-the-bottleneck-architecture.md (hunk 3 above)
Comment (mooreds) [reply site/_posts/2020-07-02-auth-and-the-bottleneck-architecture.md]: Clarified.

File: site/_posts/2020-07-02-auth-and-the-bottleneck-architecture.md (hunk 1 above)
Comment (mooreds) [reply site/_posts/2020-07-02-auth-and-the-bottleneck-architecture.md]: Donesky.

**Pull request merged**